code/index.html

```diff
@@ -1,12 +1,38 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-    <title>Project 4: Pizzabot</title>
-  </head>
-  <body>
-    <h1>This is your Pizza Bot</h1>
-  </body>
-</html>
+ <head>
+   <meta charset="UTF-8" />
+   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+   <link rel="shortcut icon" href="img/favicon.png" type="image/x-icon">
+   <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+   <link href="https://fonts.googleapis.com/css?family=Pacifico&display=swap" rel="stylesheet">
+   <link rel="stylesheet" href="style.css">
+   <title>PIZZA BOT</title>
+   <link href="https://fonts.googleapis.com/css?family=Playfair+Display&display=swap" rel="stylesheet">
+   <link href="https://fonts.googleapis.com/css?family=Abril+Fatface|Playfair+Display&display=swap" rel="stylesheet">
```

Comment:
Tip! On Google fonts you can choose multiple fonts so you only get one `<link>` to insert here instead of two